gax/src/main/java/com/google/api/gax/rpc/TransportChannelProvider.java

```diff
@@ -70,7 +70,7 @@
    *
    * <p>This method should only be called if {@link #needsExecutor()} returns true.
    */
-  TransportChannelProvider withExecutor(ScheduledExecutorService executor);
+  TransportChannelProvider withExecutor(ExecutorService executor);
```

Comment:
I think this is a breaking api change. I think you need to create a new overload instead. Similar to:
https://github.com/googleapis/gax-java/pull/869/files#diff-fb40817bacd2e653d64546666f7d9911R69-R78

Reply:
Done, thanks for the heads up.